Makefile

```diff
@@ -23,33 +29,38 @@ console:
 db:
 	@$(BASH_DEV) "bundle exec rake db:setup db:migrate"
 
+.PHONY: down
+down:
+	@$(COMPOSE_DEV) down
+
 .PHONY: guard
-guard: db
+guard:
 	@$(BASH_DEV) "bundle exec guard"
 
 .PHONY: lint
-lint: db
+lint:
 	@$(BASH_DEV) "bundle exec rake lint"
 
+.PHONY: migrate
+migrate:
+	@$(BASH_TEST) "bundle exec rake db:migrate"
+
+.PHONY: rebuild
+rebuild: down
+	@$(COMPOSE_DEV) build
+
 .PHONY: security
-security: db
+security:
 	@$(BASH_DEV) "bundle exec rake security"
 
+.PHONY: server
+server:
+	@$(BASH_DEV) "rm -f tmp/pids/server.pid && bundle exec rails server"
+
 .PHONY: spec
-spec: db
+spec:
```

Comment:
doesn't `spec` need a db?

Reply:
trying to limit when db runs, locally you wouldn't run migrate before every rspec run, IMO as long as it runs on setup steps (build, up) and ci it should be sufficient